### ISSUE TITLE
Gmail

### ISSUE THREAD

**Issue by Trevorsig** [kgretzky/evilginx2#674]
has anyone successfully bypass gmail?

**Comment by 456478** [comment]
yes, I did. it was patched today however.

my method: https://github.com/berstend/puppeteer-extra/issues/578
google's response: https://github.com/berstend/puppeteer-extra/issues/578

**Comment by Trevorsig** [comment]
@456478 whats your telegram or email?

**Comment by rickyrik001** [comment]
@Trevorsig  yes ,i can help you with that, contact me via email rickyrik@protonmail.com
telegram =@Ricky_rik1

**Comment by 456478** [comment]
@Trevorsig good luck getting scammed.

**Comment by 456478** [comment]
you have to decode the base64 string to get the email. https://www.base64decode.org/

anybody readily trying to get in touch with you here is 100% scamming you.

**Comment by klumren1** [comment]
> @Trevorsig yes ,i can help you with that, contact me via email [rickyrik@protonmail.com](mailto:rickyrik@protonmail.com)
> telegram =@Ricky_rik1

This guys a ripper be careful

**Comment by rickyrik001** [comment]
@Klumren1 buddy I can't help you no one can ,if you don't know how to use the tool !!

**Comment by klumren1** [comment]
This is cap ur a dirty ripper don’t come lie anyone that need anything cookies I recommend https://t.me/nakedpages before this guy, don’t waste your time or money with him

**Comment by rickyrik001** [comment]
@klumren1 Bro I told you again ,first learn how to use evilginx you don't know even a single command how you ran the tool ? and people I worked with know who I am ,so I don't need your certificate to proof anything :)

**Comment by klumren1** [comment]
Lies.

**Comment by klumren1** [comment]
![C2B5D180-0DFB-41D3-804D-4EA755049AC1](https://user-images.githubusercontent.com/112618432/191245505-2c5df7fe-72c9-4046-a047-00d0787d4ef4.jpeg)

**Comment by rickyrik001** [comment]
![Uploading Screenshot_20220920-164311_1.png…]()

**Comment by mrviularry** [comment]
From what i have seen, @klumren1 does not know how to use evilginx, @rickyrik001 has the best gmail or google evilginx i have ever seen.

**Comment by mrviularry** [comment]
Stop saying stuffs and learn how to use it.

**Comment by mrviularry** [comment]
@rickyrik001 is not a ripper

**Comment by rickyrik001** [comment]
@klumren1 i was trying to help you from last 3 weeks but you are so dump that you didn't know even basic commands of linux and you don't know how to even use evilginx. 
![screenshot](https://user-images.githubusercontent.com/89962590/191366328-6014402d-1a3f-4e0a-81bd-52332c37b007.png)

**Comment by klumren1** [comment]
Apologies misunderstanding @rickyrik001 fixed issue
